pkgs/development/tools/build-managers/scons/default.nix

```diff
@@ -1,28 +1,27 @@
 { lib, fetchFromGitHub, python3 }:
-
-let
+python3.pkgs.buildPythonApplication rec {
```

Comment:
1. are we finalAttrs yet?
2. python3.pkgs should be changed to python3Packages - or someone will come from the depths of some gehenna to complain about splicing, cross compiling and packages breaking on MIPS32.

Reply:
1. No
2. Yes